pylot/control/mpc/utils.py

```diff
@@ -281,6 +281,17 @@ def calc_yaw(self, s):
         yaw = np.arctan2(dy, dx)
         return yaw
 
+    def find_s(self, x, y, s0=0):
```

Comment:
Can you please comment the method?

Reply:
Sounds good, will also refactor the rest of this class' documentation.